specification/managementgroups/resource-manager/Microsoft.Management/stable/2020-02-01/management.JSON

```diff
@@ -0,0 +1,2000 @@
+        },
+        "updatedBy": {
+          "type": "string",
+          "description": "The identity of the principal or process that updated the object."
+        },
+        "parent": {
+          "title": "Parent",
+          "$ref": "#/definitions/ParentGroupInfo"
+        }
+      }
+    },
+    "ManagementGroupChildInfo": {
+      "description": "The child information of a management group.",
+      "properties": {
+        "type": {
+          "title": "The type of child resource.",
+          "$ref": "#/definitions/ManagementGroupChildType",
+          "description": "The fully qualified resource type which includes provider namespace (e.g. Microsoft.Management/managementGroups)"
+        },
+        "id": {
+          "type": "string",
+          "description": "The fully qualified ID for the child resource (management group or subscription).  For example, /providers/Microsoft.Management/managementGroups/0000000-0000-0000-0000-000000000000"
+        },
+        "name": {
+          "type": "string",
+          "description": "The name of the child entity."
+        },
+        "displayName": {
+          "type": "string",
+          "description": "The friendly name of the child resource."
+        },
+        "roles": {
+          "description": "The roles definitions associated with the management group.",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "children": {
+          "description": "The list of children.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ManagementGroupChildInfo"
+          }
+        }
+      }
+    },
+    "ManagementGroupPathElement": {
+      "description": "A path element of a management group ancestors.",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "The name of the group."
+        },
+        "displayName": {
+          "type": "string",
+          "description": "The friendly name of the group."
+        }
+      }
+    },
+    "ParentGroupInfo": {
+      "description": "(Optional) The ID of the parent management group.",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "The fully qualified ID for the parent management group.  For example, /providers/Microsoft.Management/managementGroups/0000000-0000-0000-0000-000000000000"
+        },
+        "name": {
+          "type": "string",
+          "description": "The name of the parent management group"
+        },
+        "displayName": {
+          "type": "string",
+          "description": "The friendly name of the parent management group."
+        }
+      }
+    },
+    "ManagementGroupChildType": {
+      "type": "string",
+      "enum": [
+        "Microsoft.Management/managementGroups",
+        "/subscriptions"
+      ],
+      "description": "The type of child resource."
+    },
+    "OperationResults": {
```

Comment:
Why did you fork OperationResults in the AzureAsyncOperationResults definition? They're almost identical.

For operation results, the status field is required (which happens to be the only difference between these definitions).

https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/Addendum.md#operation-resource-format

Reply:
These are 2 different async contracts because we are returning 2 different async operation headers from our APIs. One is for the location header and one is for the Azure-AsyncOperation header. We've had a long discussion internally with Tian about this and we've made these changes to respect both RPCs.